website/docs/superquickstart_web.md

```diff
@@ -0,0 +1,101 @@
+---
+id: superquickstart_web
+title: Super-quick start guide for JBrowse web
+toplevel: true
+---
+
+This is a quick overview to get started running JBrowse 2 from scratch using the command line. It is
+helpful if you have some familiarity with the command line in general to follow
```

Comment:
I'd maybe add familiarity with the command line and common bioinformatics CLI tools